ncm-sysctl/src/main/perl/sysctl.pm

```diff
@@ -94,13 +91,16 @@ sub Configure {
     # execute /sbin/sysctl -p if any change made to sysctl configuration file
     #
     if ( $changes ) {
-	unless (LC::Process::trun(300,"$sysctl_exe", "-p")) {
-	    $self->error('Failed to load sysctl settings from $configFile');
-	    return;
-	}
+        $self->verbose("Changes made to $configFile, running sysctl on it");
+        my $cmd = CAF::Process->new([$sysctl_exe, '-p', $configFile],
```

Comment:
It's pulled from the profile, so cannot  be a constant without breaking backwards compatibility.